resource_tracker/forms/resource_form.py

```diff
@@ -0,0 +1,97 @@
+    ResourceTextAttribute, ResourceGroupTextAttributeDefinition
+from service_catalog.models import Instance
+
+
+class ResourceForm(ModelForm):
+    class Meta:
+        model = Resource
+        fields = ["name", "service_catalog_instance"]
+
+    name = forms.CharField(label="Name",
+                           required=True,
+                           widget=forms.TextInput(attrs={'class': 'form-control'}))
+
+    service_catalog_instance = forms.ModelChoiceField(queryset=Instance.objects.all(),
+                                                      label="Service catalog instance",
+                                                      required=False,
+                                                      widget=forms.Select(attrs={'class': 'form-control'}))
+
+    def __init__(self, *args, **kwargs):
+        resource_group_id = kwargs.pop('resource_group_id', None)
+        self.resource_group = ResourceGroup.objects.get(id=resource_group_id)
+        super(ResourceForm, self).__init__(*args, **kwargs)
+        self._newly_created = kwargs.get('instance') is None
+        self.attributes_name_list = list()
+        self.text_attributes_name_list = list()
+
+        for attribute in self.resource_group.attribute_definitions.all():
+            initial_value = None
+            if self.instance is not None:
+                try:
+                    initial_value = ResourceAttribute.objects.get(resource=self.instance,
+                                                                  attribute_type=attribute)
+                except ResourceAttribute.DoesNotExist:
+                    pass
+            new_field = forms.IntegerField(label=attribute.name,
+                                           min_value=0,
+                                           required=False,
+                                           initial=initial_value,
+                                           help_text="Please fill with positive integer",
+                                           widget=forms.TextInput(attrs={'class': 'form-control'}))
+            self.fields[attribute.name] = new_field
+            self.attributes_name_list.append(attribute.name)
+        for text_attribute in self.resource_group.text_attribute_definitions.all():
+            initial_value = None
+            if self.instance is not None:
+                try:
+                    initial_value = ResourceTextAttribute.objects.get(resource=self.instance,
+                                                                      text_attribute_type=text_attribute)
+                except ResourceTextAttribute.DoesNotExist:
+                    pass
+            new_field = forms.CharField(label=text_attribute.name,
+                                        max_length=500,
+                                        required=False,
+                                        initial=initial_value,
+                                        help_text="Text field",
```

Comment:
The help text here should be set by the user. (non mandatory field on attribute creation)